script/bootstrap

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 set -e
 
-npm install
+npm ci # clean install
```

Comment:
Huh. I always read this as "Prepare Node for a CI (Continuous Integration) environment". Though "clean install" makes more sense.